src/uwtools/templater.py

```diff
@@ -68,17 +69,36 @@ def parse_args(argv):
         action='store_true',
         help='If provided, print a list of required configuration settings to stdout',
         )
+    parser.add_argument(
+        '-v', '--verbose',
+        action='store_true',
+        help='If provided, print all logging messages.',
+        )
+    parser.add_argument(
+        '-q', '--quiet',
+        action='store_true',
+        help='If provided, print no logging messages',
+        )
     return parser.parse_args(argv)
 
 def set_template(argv):
     '''Main section for rendering and writing a template file'''
-
     user_args = parse_args(argv)
-    print("Running script templater.py with args:n", f"{('-' * 70)}\n{('-' * 70)}")
+
+    logfile = os.path.join(os.path.dirname(__file__), "templater.log")
+    log = Logger(level='info', logfile_path=logfile)
+    if user_args.verbose:
+        log = Logger(level='debug', logfile_path=logfile, colored_log=True)
+        log.debug('Finished setting up debug file logging in %(logfile)s')
+    elif user_args.quiet:
+        log.propagate = False
+
+
+    log.info('Running script templater.py with args:n '+('-' * 70)+'\n'+('-' * 70))
```

Comment:
There is a typo in this line. there should be a newline after `args:`.

Reply:
Yep, I marked this as #127 as it's in the current version, but I can include it in this PR